osbs/conf.py

```diff
@@ -426,3 +427,34 @@ def get_oauth2_token(self):
     def get_reactor_config_secret(self):
         return self._get_value("reactor_config_secret", self.conf_section,
                                "reactor_config_secret")
+
+    def get_client_config_secret(self):
+        return self._get_value("client_config_secret", self.conf_section,
+                               "client_config_secret")
+
+    def get_token_secrets(self):
+        value = self._get_value("token_secrets", self.conf_section,
+                                "token_secrets")
+        token_dict = {}
+        will_raise = False
+        if value:
+            for pairs in value.split():
+                if pairs.count(':') >= 1:
+                    logger.error("token_secret can't contain multiple colons : %s", pairs)
+                    will_raise = True
+                    continue
+                pair = pairs.split(':')
+
+                if len(pair) > 1:
```

Comment:
What's this test for? It seems to be what causes 'foo:x' and 'foo:' to be treated like 'foo'.